impl/src/main/java/io/kroxylicious/testing/kafka/common/Utils.java

```diff
@@ -151,6 +153,14 @@ public static void awaitExpectedBrokerCountInClusterViaTopic(Map<String, Object>
                         final Boolean isQuorate = promise.getNow(false);
                         if (isQuorate) {
                             admin.deleteTopics(Set.of(CONSISTENCY_TEST));
+                            for (int i = 0; i < admin.describeCluster().nodes().get().size(); i++) {
```

Comment:
I think we should have timeouts on the `get` call. I'm also kinda wary of doing a remote RPC on every loop.

Reply:
I have removed the loop. It does not make sense if we remove the topics in the cluster and wait for the removal for each broker